Source/WebKit/WebProcess/WebPage/WebPage.cpp

```diff
@@ -6000,11 +6000,13 @@ void WebPage::effectiveAppearanceDidChange(bool useDarkAppearance, bool useEleva
 void WebPage::freezeLayerTreeDueToSwipeAnimation()
 {
     freezeLayerTree(LayerTreeFreezeReason::SwipeAnimation);
+    corePage()->setIsInSwipeAnimation(true);
```

Comment:
This doesn't feel like quite the right place to hook into, or we should rename this function (and the IPC) to be `swipeAnimationWillStart()` or something.

Reply:
That makes sense, I tried to give some better naming for swipe start/end.